src/components/dashboard/shared/VerticalTimeline.js

```diff
@@ -16,24 +16,42 @@ export default class VerticalTimeline extends Component {
   sortItems = memoize((items) => items.sort(((a, b) => moment(b.date) - moment(a.date))));
 
   handleViewMore = () => {
-    const { items, viewMoreCount } = this.props;
+    const { items, viewCount } = this.props;
     const { displayCount  } = this.state;
-    let newDisplayCount = displayCount + viewMoreCount;
+    let newDisplayCount = displayCount + viewCount;
     if (newDisplayCount > items.length) { newDisplayCount = items.length; }
 
     this.setState({ displayCount: newDisplayCount });
   }
 
+  handleViewLess = () => {
+    const { viewCount } = this.props;
+    const { displayCount  } = this.state;
+    let newDisplayCount = displayCount - viewCount;
+    if (newDisplayCount <= viewCount) { newDisplayCount = viewCount; }
```

Comment:
After giving this some more thought, I think it makes more sense to have the minimum display be equal to `initialDisplayCount` instead of `viewCount`. So I would change this line to be `if (newDisplayCount < initialDisplayCount) { newDisplayCount = initialDisplayCount; }` and add the prop name to line 28.